section2/exercises/if-statements.js

```diff
@@ -43,7 +43,15 @@ Log to the console the appropriate age range
 
 var dogAge = 3;
 // Write your conditional here
-
+if (dogAge <= 2) {
```

Comment:
Very clean style and syntax (use of spaces, white space, etc) of your if/else 🙌

Since the prompt only asked to determine if the dog was a puppy, adult, or elderly, we don't need the final else! Or, the elderly could fall into the "else" since anything at or under 9 would already have been identified and stopped in a previous block